src/IdeaStatiCa.OpenModel.VersioningService/Configuration/IConfigurationStepService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using IdeaStatiCa.OpenModel.VersioningService.VersionSteps;
+
+namespace IdeaStatiCa.OpenModel.VersioningService.Configuration
+{
+	public interface IConfigurationStepService
```

Comment:
Add the description of the responsibility of the class

Reply:
added